### PR TITLE
fix: debank & zerion api on multiple networks

### LINKED ISSUE #3598: [Bug] Transaction List & EVM compatible networks
## Catalogue

- [ ] UI/UX
- [ ] Mask Core Feature
- [x] Mask Plugin Feature
- [ ] Security/Cryptography Feature
- [ ] Other
  - /_List it here_/

## Part 1: Introduction and Background

Some EVM compatible networks' transaction list is not working.

### PR DESCRIPTION
**Pull request by guanbinrui**
closes #3598.

### CODE REVIEW

File: packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
     ERC721TokenAssetDetailed,
     ChainId,
     CollectibleProvider,
-    resolveCollectibleLink,
 } from '@masknet/web3-shared'
 import { memo } from 'react'
 import { Link, makeStyles, Typography } from '@material-ui/core'
 import { MaskColorVar } from '@masknet/theme'
 import { CollectiblePlaceholder } from '../CollectiblePlaceHolder'
+import { resolveCollectibleLink } from '../../../../../../maskbook/src/plugins/Wallet/pipes'
```
Comment (nuanyang233) [inline packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx:11]: It is not allowed to import from the maskbook package into the dashboard package

File: packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
-import {
+import type {
     ERC1155TokenAssetDetailed,
     ERC721TokenAssetDetailed,
     ChainId,
     CollectibleProvider,
-    resolveCollectibleLink,
```
Comment (nuanyang233) [inline packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx:6]: Why not use it?

File: packages/web3-shared/src/pipes/index.ts [hunk 3]
```diff
@@ -111,29 +102,6 @@ export function resolveIPFSLink(ipfs: string): string {
     return `https://ipfs.fleek.co/ipfs/${ipfs}`
 }
 
-export function resolveCollectibleProviderLink(chainId: ChainId, provider: CollectibleProvider) {
```
Comment (nuanyang233) [inline packages/web3-shared/src/pipes/index.ts:114]: Why move this function?

File: packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx:11]: Yes, please move it to some shared package.

Comment (UncleBill) [reply packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx:11]: `resolveCollectibleLink` doesn't seem to rely on any context from `maskbook`, maybe we do the refactoring of moving all plugins into an individual package later.

Comment (nuanyang233) [reply packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx:11]: `resolveCollectibleLink` is pipe. IMO this move to a utils package like `web3-shared` would be better.

Comment (guanbinrui) [reply packages/dashboard/src/pages/Wallets/components/CollectibleCard/index.tsx]: It looks like we separate plugin code into different packages. It's hard to maintain.

**Review by septs — APPROVED** [review]

**Review by UncleBill — APPROVED** [review]

**Review by nuanyang233 — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**